CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ Please see [CONTRIBUTING.md](https://github.com/cucumber/cucumber/blob/master/CO
   ([#1136](https://github.com/cucumber/cucumber-js/issues/1136)
   [#1721](https://github.com/cucumber/cucumber-js/pull/1721))
 - Support for configuration to be objects instead of argv strings, and for configuration files in ESM and JSON formats ([#1952](https://github.com/cucumber/cucumber-js/pull/1952))
+- New API for running Cucumber programmatically ([#1955](https://github.com/cucumber/cucumber-js/pull/1955))
```

Comment:
This is a big thing. I would introduce it a little bit, and at least add a link pointing to the api documentation.

Reply:
Yes good point. We'll do a thorough release note for 8.0.0 but this is a bit terse.